test/should-run/throw-in-catch.js

```diff
@@ -2,9 +2,9 @@ const assert = require('assert');
 let a = 0;
 try {
   try {
-    foo + 1;
+    throw "x";
```

Comment:
The point of this was it's not an explicit `throw`, but a runtime error. Does this need to be changed?